action.yml

```
@@ -14,11 +14,28 @@ inputs:
     description: "Name of the Google cloud storage bucket to upload the artifacts to"
     required: false
     default: ".deploy"
+  sha:
+    description: |
+      Sha used to checkout artifacts. 
+      If sha is not provided even trigger is matched to sha following way.
```

unsure suggestion: ?
```suggestion
      If `sha` is not provided, then the triggering event is matched to the SHA the following way.
```